packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.css

```diff
@@ -96,6 +96,12 @@
   cursor: default;
 }
 
+.end-item-icon {
+  width: 14px;
+  height: 14px;
+  fill: var(--neutral-600);
```

Comment:
Could we make this configurable? Maybe just something along the lines of `--scene-tree-secondary-font-color` (or something similar) with a default of `--neutral-600`? That might also make it possible to have the CSS definition be reusable for any future "secondary" icons we may add

Reply:
Good idea - updated!